ui/lang/he-IL.json

```diff
@@ -1,582 +1,597 @@
 {
-  "+C8Jng": "התאם את הטופס ל{bucketName}.",
-  "+R1gxN": "הוסף קווים מנחים",
+  "+C8Jng": "התאם אישית את הטופס עבור {bucketName} למילוי.",
+  "+R1gxN": "הוסף קו מנחה",
   "+SgU+0": "IBAN",
-  "+T1/GD": " ",
-  "+a+2ug": "משתתפים",
-  "+fQNBX": "מאזן",
-  "+fkK2J": "הצורך הנ\"ל עדיין לא מכיל משימות. \nניתן להוסיף משימות על ידי לחיצה למעלה או ל<a> לחץ כאן</a> על מנת לצפות בה ישירות",
-  "+n0CL7": "אחריות",
-  "/0TOL5": "סכום",
-  "/AmWqb": "+ תמונת קאבר",
-  "/LAG1H": "הפוך ללא מוכן ללבבות",
-  "/T+mWk": "Remove moderator",
-  "/U0AO2": "כל התגיות",
-  "/h2MHX": "ברוכים וברוכות הבאות ל{bucketName}!",
+  "+T1/GD": "תחילה עליך לחבר את הפרויקט הזה ל-Stripe",
+  "+a+2ug": "חברים",
+  "+fQNBX": "יתרת משתמש",
+  "+fkK2J": "צורך זה אינו מכיל אחריות עדיין.  <a>לחץ כאן</a> כדי לראות את הצורך ישירות",
+  "+n0CL7": "אַחֲרָיוּת",
+  "/0TOL5": "כמות",
+  "/AmWqb": "תמונת שער",
+  "/LAG1H": "סמן כלא מוכן",
+  "/T+mWk": "הסר את המנחה",
+  "/U0AO2": "כל התגים",
+  "/h2MHX": "ברוכים הבאים אל {bucketName}!",
   "00LcfG": "טען עוד",
-  "01J6C+": "Email column missing in CSV file",
-  "0BUTMv": "חיפוש...",
-  "0HT+Ib": "ארכיון",
-  "0KKXrH": "תקציב",
-  "0WYWwl": "האם אתה בטוח שברצונך למחוק את התגית הזאת?",
-  "0b3BWi": "Add Opencollective token",
-  "0ci3jg": "רשימת מיילים מופרדת באמצעות פסיק",
-  "0pcuLv": "Funds available",
-  "0qsdJO": "הצג את הנחיות המימון",
-  "0vJAZN": "תודה לך! התגובה שלך נשמרה.",
-  "1EYCdR": "תגיות",
-  "1Ia8qN": "Processing",
-  "1KCgzV": "ה{bucketName} סומן כעובר על ההנחיות של {title} עם ההערה: {comment}",
-  "1NbajR": "All Buckets",
-  "1ZgrhW": "כלל",
-  "1dQvV0": "תרומה מינימלית",
-  "1gZbl7": "חלום חדש בשם {bucketName} נוצר",
+  "01J6C+": "עמודת דוא\"ל חסרה בקובץ CSV",
+  "0BUTMv": "לחפש...",
+  "0HT+Ib": "בארכיון",
+  "0KKXrH": "תַקצִיב",
+  "0WYWwl": "האם אתה בטוח שברצונך למחוק את התג הזה לצמיתות?",
+  "0b3BWi": "הוסף אסימון Opencollective",
+  "0ci3jg": "הודעות אימייל מופרדות בפסיק",
+  "0pcuLv": "לבבות זמינים",
+  "0qsdJO": "ראה הנחיות ללב",
+  "0vJAZN": "תודה! ",
```

Comment:
The translation here reduced the context, the original was something about comments.

Reply:
Prior translation was:

תודה לך! התגובה שלך נשמרה

Reply:
Specifically talking about "0vJAZN"